src/Sulu/Bundle/ContentBundle/Resources/public/js/validation/types/resourceLocator.js

```diff
@@ -39,7 +39,7 @@ define([
                 },
 
                 needsValidation: function() {
-                    return this.$el.find('input').length > 0;
+                    return this.$el.data('validationRequired') || false;
```

Comment:
I don't think that this is correct... Just because the field is not required, it doesn't mean that it shouldn't be validated. So if the field is not required you are suddenly allowed to add stuff like umlauts?